### ISSUE TITLE
[SF3] Hopseed tree grows on wrong y value

### ISSUE THREAD

**Issue by EnderofGames** [progwml6/Natura#384]
Natura version: 4.1.0.56
SkyFactory version: 3.0.10
Minecraft version: 1.10.2

To preface: This is not a pressing issue, and I have yet to update to the newest version of SF3 because my server admin has not been online. I am confident that this issue hasn't been fixed from what I have seen in the issues and the changelog. 

I am aware of some limitation of the _hopseed_, _tigerwood_, _maple_, and _willow_ trees only being growable at certain y-values. (This does not seem to affect _eucalyptus_, _sakura_, or the nether trees) I do not know what this value is, but I can say I have not changed it since download, nor has my server admin; though I can reproduce this glitch in both SSP and SMP.

When I plant a "restricted sapling" at y=46, it will not grow, but while setting up a farm I noticed a misplaced grown _hopseed_ tree. Note that the farming station present is not powered not fed saplings, so it is not the source of this error.

![2017-07-20_22 22 35](https://user-images.githubusercontent.com/17417666/28449944-4f051b2a-6da1-11e7-8e7b-26b5fc54a6af.png)

The higher platform is at y=66. I removed the tree and waited, and the _hopseed_ tree still grew at y=66, no 'twerking', no bonemeal, same sapling. I tried putting stone in between, I tried making multiple levels of dirt. The _hopseed_ tree will always grow at the highest y value, even if it is currently at a growable y value. If there are dirt spaces higher that are too short for _hopseed_ tree growth, either limited by blocks or the world limit, it will grow at the highest 'legal' point. 

Once I grow a tree, I cannot seem to reuse the sapling until I remove the tree. I can sometimes grow _hopseed_ trees when dirt is above them, but I cannot tell why it works sometimes, and why I can't grow trees at 'legal heights' at other times.

**Comment by alexbegt** [comment]
Inside the config file for natura (natura.cfg), change the value of "Sea level" to something lower than 64, Please reply back if this does not fix the issue.

**Comment by alexbegt** [comment]
Hi, I am wondering if this bug still occurs?

**Comment by EnderofGames** [comment]
Sorry for the delay, with the SF3 3.0.13 update I had issues, and my SMP server is down for a while. I tried to replicate with SSP only. 

So far, no improvement. 

I changed both the "Flat Sea level" and "Sea level" to 1, and no change. 

![2017-08-30_18 28 36](https://user-images.githubusercontent.com/17417666/29901102-5c7dff42-8db3-11e7-9fd1-910171ed4b62.png)

I then changed both values to 100, and the bug still occured between 74 and 89. (74 being the sapling y-value, 89 being max leaf height after planting)

![2017-08-30_18 30 11](https://user-images.githubusercontent.com/17417666/29901109-63239032-8db3-11e7-8d03-3d2926a2ff48.png)
